JapaneseFilter/sections/css_extended.txt

```diff
@@ -7,6 +7,8 @@ animeanime.jp,autocar.jp,getnavi.jp,the-uranai.jp#?#._popIn_infinite_page > li:h
 cafend.net,chosunonline.com,taishu.jp#?##_popIn_recommend li:has(._popIn_recommend_article_ad)
 honcierge.jp,otonasalone.jp#?#._popIn_infinite_page:has(._popIn_recommend_article_ad)
 cinemacafe.net#?#._popIn_infinite_page:has(._popIn_recommend_article_ad_reserved)
+!
```

Comment:
Is this separation required?

Reply:
Not really, just thought it may be visually helpful as we may add popIn entries in future - I found similar separation in other AG lists.

Reply:
I mean that rules are grouped, and new `!` is between that rules
https://github.com/AdguardTeam/AdguardFilters/blob/dbe8fa481ee63bfa31193a766250ee6647345041/JapaneseFilter/sections/css_extended.txt#L4-L16

Reply:
Your group mean `!` under `mindhack2ch.com#?#.article-post-bottom > .rss-article:has(> .rss-article-item)`? I rather don't understand why `!` is there.

Reply:
I mean I don't see any qualitative difference above and below the `!`. I guess it can be removed than `!` under popIn group I made.